src/docs/faq.md

```diff
@@ -1,88 +1,101 @@
+<br>
 
-#### I'm using your UEFI firmware, installed my OS [Linux], and it still boots to the EFI shell - what do I do?
-* Sounds like your Linux distro doesn't install the EFI bootloader in the default location - no worries, it's an easy fix:
-  * Type `exit` to return to the UEFI settings menu, then select Boot Maintenance Manager. From there, select Boot From File, then navigate to and boot from `/EFI/[distro name]/grubx64.efi` (where [distro name] will be fedora, arch, debian, etc). Once your OS is booted, open a terminal/shell, and type the following (observing case):
+## Linux Questions
 
-   * `sudo su`
-   * `mkdir -p /boot/efi/EFI/BOOT`
-   * `cp /boot/efi/EFI/[distro name from above]/grubx64.efi /boot/efi/EFI/BOOT/BOOTX64.efi`
+### How come audio isn't working on Ubuntu / Ubuntu forks?
+* Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to ubuntu.
 
-   * Then reboot to test. What we're doing is copying the grub EFI boot stub from the OS installed location to the location the firmware is expecting (/EFI/BOOT/BOOTX64.efi) on the EFI system partition (ESP), which most (Debian/Ubuntu-based?) distros will mount at /boot/efi. You may need to adjust slightly for your distro, but these instructions should work in most cases.
+### What Linux distros are recommended?
+* See [this page](installing/installing-linux.md).
 
-#### I'm running Legacy boot firmware now - can I switch to the UEFI firmware?
-* You can, but your existing OS install won't boot. You'll need to either reinstall the OS (often the easiest course of action), or you can attempt to migrate your existing install, but this isn't ideal and should be considered unsupported.
+### How can I get audio working under Linux?
+* Please see [this github repo](https://github.com/WeirdTreeThing/chromebook-linux-audio)
 
-#### Help! I can't exit Developer Mode
-* If you're trying to exit Developer Mode but get the error `WARNING: TONORM prohibited by GBB_FORCE_DEV_SWITCH_ON` or find the `Return to secure mode` button missing, you need to reset your GBB flags. You can do that by running [MrChromebox's firmware utility script](https://mrchromebox.tech/#fwscript) and selecting `3) Set Boot Options (GBB flags)`, followed by `5) Reset to factory default`. 
-  * Then exit developer mode by running `sudo crossystem disable_dev_request=1; reboot`
+### How do I get my top row keys on Linux to act like how they did on chromeOS?
+* Follow the instructions [in this github repo](https://github.com/WeirdTreeThing/cros-keyboard-map).
 
-#### Can I modify the fan speed?
-* Yes, using ectool. To set the fan to auto, run: `ectool autofanctrl`. To specify a percentage, run `ectool fanduty [0-100]`.
+<br>
+
+## Hackintosh questions
 
-#### Can I overclock the fan?
+### Can I hackintosh my Celeron/Pentium CPU?
```

Comment:
A FAQ question for AMD chips should be added tbh

Reply:
If there are frequently asked questions about AMD

Reply:
There are, it's been asked a few times